simpa_examples/minimal_optical_simulation_uniform_cube.py

```diff
@@ -0,0 +1,96 @@
+# SPDX-FileCopyrightText: 2021 Division of Intelligent Medical Systems, DKFZ
+# SPDX-FileCopyrightText: 2021 Janek Groehl
+# SPDX-License-Identifier: MIT
+
+# SPDX-FileCopyrightText: 2021 Division of Intelligent Medical Systems, DKFZ
+# SPDX-License-Identifier: MIT
+
+
+from simpa import Tags
+import simpa as sp
+import numpy as np
+
+# FIXME temporary workaround for newest Intel architectures
+import os
+os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
+
+# TODO: Please make sure that you have set the correct path to MCX binary and SAVE_PATH in the file path_config.env
+#  located in the simpa_examples directory
+path_manager = sp.PathManager()
+
+VOLUME_TRANSDUCER_DIM_IN_MM = 60
+VOLUME_PLANAR_DIM_IN_MM = 30
+VOLUME_HEIGHT_IN_MM = 60
+SPACING = 0.5
+RANDOM_SEED = 471
+VOLUME_NAME = "MyVolumeName_"+str(RANDOM_SEED)
+SAVE_REFLECTANCE = True
+SAVE_PHOTON_DIRECTION = False
+
+# If VISUALIZE is set to True, the simulation result will be plotted
+VISUALIZE = True
+
+
+def create_example_tissue():
+    """
+    This is a very simple example script of how to create a tissue definition.
+    It contains a muscular background, an epidermis layer on top of the muscles
```

Comment:
I can only see a background, no muscles and epidermis :joy:

Reply:
This is on purpose to simplify the script.

Reply:
Yes and it makes sense to simplify the script. But then the comment would also need to be update, otherwise readers are confused where the epidermis and vessels are, right?

Reply:
I see what you mean, fixed it now